pkgs/desktops/plasma-5/3rdparty/addons/krunner-symbols.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv
+, cmake, fetchFromGitHub, extra-cmake-modules
+, qtbase, wrapQtAppsHook, ki18n, kdelibs4support, krunner
+}:
+stdenv.mkDerivation rec {
+  pname = "krunner-symbols";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "domschrei";
+    repo = "krunner-symbols";
+    rev = version;
+    sha256 = "sha256-YsoZdPTWpk3/YERwerrVEcaf2IfGVJwpq32onhP8Exo=";
+  };
+
+  buildInputs = [ qtbase ki18n kdelibs4support krunner ];
+  nativeBuildInputs = [ cmake wrapQtAppsHook extra-cmake-modules ];
+
+  prePatch = ''
+    sed -i 's,/usr/share/config/krunner-symbol,'$out'/share/config/krunner-symbol,' symbols.cpp
+
+    sed -i 's,LOCATION_PLUGIN,KDE_INSTALL_PLUGINDIR,' CMakeLists.txt
+    sed -i 's,LOCATION_DESKTOP,KDE_INSTALL_KSERVICES5DIR,' CMakeLists.txt
+  '';
+
+  cmakeFlags = [ "-DLOCATION_CONFIG=share/config" ];
+
+  meta = with lib; {
+    description = "A little krunner plugin (Plasma 5) to retrieve unicode symbols, or any other string, based on a corresponding keyword";
+    homepage = "https://github.com/domschrei/krunner-symbols";
+    license = licenses.gpl3;
```

Comment:
Fairly sure `licenses.gpl3` is deprecated, I believe you're supposed to use `licenses.gpl3Only` or `licenses.gpl3Plus` now.